decentraland/unity-renderer
# fix: Update Terms of Service in Explorer

## What does this PR change?
Fixes #2098 

Terms of Service displayed during the sign up process of the Explorer were not updated.

**Acceptance Criteria**
Explorer ToS should contain the official https://decentraland.org/terms/

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=fix/update-terms-of-service-in-explorer
2. Enter as guest.
3. Notice the ToS are updated.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
